plugin/src/leiningen/cljs_lambda/aws.clj

```diff
@@ -39,9 +44,9 @@
                               "]"])}))))
 
 (defn ->cli-args [m & [positional {:keys [preserve-names?]}]]
-  (let [m    (cond-> (merge (meta-config) (update-vpc-value m))
+  (let [m    (cond-> (merge (meta-config) (-> m (update-vpc-value) (update-dead-letter-value)))
                (not preserve-names?)
-               (set/rename-keys {:name :function-name :vpc :vpc-config}))
```

Comment:
@zrzka Now that we have a couple of the these transformations, how would you feel if the `args` expression below did something like:

```clojure
(for [[k v] m :let [v (arg->cli k v)]]
  [k v]))
```

Where `arg->cli` (or whatever, that's probably not a great name) is a multimethod dispatching on the first arg, with a default clause which does the `(if (keyword? v) ...` bit that's currently below - and method definitions for `:vpc` and `:dead-letter`.  That'd allow us to put a bit of distance between the cli stuff and the argument-specific transformations.